controllers/token.go

```diff
@@ -340,7 +340,7 @@ func (c *ApiController) IntrospectToken() {
 
 	if application.TokenFormat == "JWT-Standard" {
 		jwtToken, err := object.ParseStandardJwtTokenByApplication(tokenValue, application)
-		if err != nil || jwtToken.Valid() != nil {
+		if err != nil {
```

Comment:
The `Valid()` method has been removed in v5. 

`jwt.ParseWithClaims` already validates the claims for us and will return `err` if there are any validation errors.

https://github.com/golang-jwt/jwt/blob/v5.2.2/parser.go#L116-L126